Dockerfile-run-tests

```diff
@@ -0,0 +1,5 @@
+FROM yara-rules
```

Comment:
Maybe its worth renaming this Docker file? Might be a bit confusing as could mean run the unit tests? Maybe something like: `Dockerfile-check-rules`

Reply:
Yeah, makes sense. I've renamed it and updated the Jenkinsfile.